front/lib/swr/assistants.ts

```diff
@@ -163,7 +165,7 @@ export function useSuggestedAgentConfigurations({
 
   const { data, error, mutate, mutateRegardlessOfQueryParams } =
     useSWRWithDefaults(key, agentConfigurationsFetcher, {
-      disabled: inCache,
+      disabled: inCache || disabled,
```

Comment:
you need to update the "isSuggestedAgentConfigurationsLoading" , should be false if disabled